pkg/odo/cli/delete/component.go

```diff
@@ -42,6 +65,29 @@ func (o *ComponentOptions) Run() error {
 
 // deleteNamedComponent deletes a component given its name
 func (o *ComponentOptions) deleteNamedComponent() error {
+	log.Info("Searching resources to delete, please wait...")
+	list, err := o.clientset.DeleteClient.ListResourcesToDelete(o.name, o.namespace)
+	if err != nil {
+		return err
+	}
+	if len(list) == 0 {
+		log.Infof("No resource found for component %s\n", o.name)
```

Comment:
IMO, it would be helpful to add namespace info to this message like we have upon successful deletion:
```suggestion
		log.Infof("No resource found for component %s in namespace %s\n", o.name, o.namespace)
```